controllers/news.controllers.js

```diff
@@ -30,8 +32,18 @@ exports.getArticleById = (req, res, next) => {
     .catch(next);
 };
 
+
+exports.getCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  selectCommentByArticleId(article_id)
```

Comment:
we need to check it the article exists as well.  As this will determine whether no articles being return indicates an article that has no comments, or an article that doesn't exist!